string name
add_furni_gui/Furni furni